### ISSUE TITLE
Noya Week 3

### ISSUE THREAD

**Issue by noyah81** [noyah81/p2ducks#35]
- back end of displaying tweets
- create a database where the inputted tweets will go into (later on the tweets will be taken out of database to be displayed on to html page)
- create database through sqlite
- create a table within the database to organize the data being inputted so it can be taken out later 
- all code on app.py for database creation

**Comment by MaxVukovich** [comment]
Good Idea for the ticket, this ticket is fundamental to the overall creation of your "twitter" website. I like the idea and using data is a great way for the creation of this ticket. I am not well versed in database types but I believe you are on the right track!

**Comment by 0404btruong** [comment]
Good work on ticket, this is necessary for all websites so it's good, maybe also do the front end for it if teammates need help. Even without that, very good ticket and idea.

**Comment by YanqiaoD** [comment]
Great work, the website looks nice and I like your idea for the ticket.

**Comment by ghost** [comment]
Solid ticket Noya. You could add in more details to specify which part of the backend you are going to work on and how the tweets are going to be displayed.

**Comment by sarahwxie** [comment]
Hey Noya! The bullet point format is really nice - it provides a readable and accessible way to view your progress. However, it would be cool to see your struggles and how you overcame them while completing this ticket.

**Comment by nivupai** [comment]
Hey Noya, nice work on getting the tweet database in. The tweet feature looks so cool now :) I know this is mainly backend work but great job on working on this, it makes a great base for our tweet feature.
